ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TutorialLiveShareLayout.kt

```diff
@@ -0,0 +1,153 @@
+    ) {
+        Spacer(modifier = Modifier.weight(1f))
+        Box(modifier = Modifier.fillMaxWidth()) {
+            Column() {
+                Spacer(modifier = Modifier.height(290.dp))
+                Spacer(modifier = Modifier.minLinesHeight(1, MaterialTheme.typography.titleLarge))
+                Spacer(modifier = Modifier.minLinesHeight(8, MaterialTheme.typography.bodyMedium))
+            }
+
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+                modifier = Modifier.fillMaxWidth()
+            ) {
+
+                Text(
+                    text = stringResource(title),
+                    style = MaterialTheme.typography.titleLarge,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.padding(horizontal = 32.dp)
+                )
+
+                Text(
+                    stringResource(body),
+                    style = MaterialTheme.typography.bodyMedium,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.padding(horizontal = 32.dp)
+                )
+
+                if (body2 != null) {
+                    Spacer(
+                        modifier = Modifier.minLinesHeight(
+                            minLines = 1,
+                            textStyle = MaterialTheme.typography.bodyMedium
+                        )
+                    )
+                    Text(
+                        stringResource(body2),
+                        style = MaterialTheme.typography.bodyMedium,
+                        textAlign = TextAlign.Center,
+                        modifier = Modifier.padding(horizontal = 32.dp)
+                    )
+                }
+                val composition by rememberLottieComposition(
+
+                    LottieCompositionSpec.RawRes(anim)
+                )
+
+                val progress by animateLottieCompositionAsState(
+                    // pass the composition created above
+                    composition,
+
+                    // Iterates Forever
+                    iterations = LottieConstants.IterateForever,
+
+                    // pass isPlaying we created above,
+                    // changing isPlaying will recompose
+                    // Lottie and pause/play
+                    isPlaying = true,
+
+                    // pass speed we created above,
+                    // changing speed will increase Lottie
+                    speed = 1f,
+
+                    // this makes animation to restart
+                    // when paused and play
+                    // pass false to continue the animation
+                    // at which is was paused
+                    restartOnPlay = false
+
+                )
+                LottieAnimation(
+                    composition,
+                    { progress },
+                    modifier = Modifier
+                        .height(290.dp)
```

Comment:
based on the xml, this height should be `R.dimen.tutorial_page_live_share_anim_height`